rott/dosutil.c

```diff
@@ -161,7 +163,7 @@ void DisplayTextSplash(byte *text, int l)
 	printf ("\033[m");
 }
 
-#if !defined(__CYGWIN__) && !defined(__MINGW32__) && !defined(__HAIKU__)
+#if !defined(__CYGWIN__) && !defined(__MINGW32__) && !defined(__HAIKU__) && !defined(_MSC_VER)
```

Comment:
Well, this excludes all supported platforms now. Maybe it's time to remove this code altogether.